buf.gen.ts-sdk.yaml

```diff
@@ -0,0 +1,24 @@
+# Learn more: https://buf.build/docs/configuration/v2/buf-gen-yaml
+version: v2
+clean: true
+plugins:
+  - local: ./ts-sdk/node_modules/.bin/protoc-gen-es
+    out: ts-sdk/src/gen/protos
+    strategy: all
+    include_imports: true
+    include_wkt: true
+    opt:
+      - target=ts
+      - json_types=true
+  - local: ./ts-sdk/bin/protoc-sdk-object.ts
+    out: ts-sdk/src/gen
+    strategy: all
+    include_imports: true
+    include_wkt: true
+    opt:
+      - target=ts
+
+inputs:
+  - module: buf.build/cosmos/cosmos-sdk
```

Comment:
do we need to restrict to smaller amount of rpc calls we actually want to use from cosmos sdk?

Reply:
there is [buf.gen.ts.yaml](https://github.com/akash-network/akash-api/blob/sdk-47/buf.gen.ts.yaml) lets use it